index.js

```diff
@@ -92,6 +92,24 @@ function isHTMLString (html) {
   return typeof html === 'string' && /(<([^>]+)>)/i.test(html)
 }
 
+/**
+ * Filters all violations by user impact
+ * @param {object} violations result of the accessibilty check by axe
+ * @param {array} impactLevels defines which impact level should be considered (e.g ['critical'])
+ * The level of impact can be "minor", "moderate", "serious", or "critical".
+ * @returns {object} violations filtered by impact level
+ */
+function filterViolations (violations, impactLevels) {
+  if (typeof violations === 'undefined') {

```

Comment:
Could we remove this and move the filter below line 126 so avoid doing this twice?

Reply:
@nickcolley Sorry for the late reply. I completely forgot about this PR. I am very happy that it has been merged, and I see that you have made the change mentioned. thx!